can-bus/json/Debug/Debug_rx.json

```
@@ -0,0 +1,3 @@
+{
+    "messages": []
+}
```

nit: missing blank line
```suggestion
}

```